airbyte-integrations/connectors/source-vitally/source_vitally/vitally.yaml

```diff
@@ -0,0 +1,107 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: ["results"]
+  requester:
+    url_base: "https://rest.vitally.io/resources/"
+    http_method: "GET"
+    authenticator:
+      type: BasicHttpAuthenticator
+      username: "{{ config['api_key'] }}"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: DefaultPaginator
+      pagination_strategy:
+        type: "CursorPagination"
+        cursor_value: "{{ response.next }}"
+        page_size: 100
+      page_size_option:
+        field_name: "limit"
+        inject_into: "request_parameter"
+      page_token_option:
+        field_name: "from"
+        inject_into: "request_parameter"
+      url_base:
+        $ref: "*ref(definitions.requester.url_base)"
+    requester:
+      $ref: "*ref(definitions.requester)"
+
+  # base stream
+  base_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+
+  # stream definitions
+  accounts_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "accounts"
+      primary_key: "id"
+      path: "/accounts"
+    retriever:
+      $ref: "*ref(definitions.base_stream.retriever)"
+      requester:
+        $ref: "*ref(definitions.requester)"
+        request_options_provider:
+          request_parameters:
+            status: "{{ config['status'] }}"
```

Comment:
This is a stream needs a special request parameter and the only one not tested. Can you try to add data to this stream?

Reply:
My bad, I already have data in this stream.

Unfortunately, I don't have any test account here as well and no free trial is available :/ 

Let me record a video to showcase the output.

Reply:
Acceptance test:
![Screenshot 2022-11-01 at 19 06 33](https://user-images.githubusercontent.com/65104164/199306158-b28fbddf-ff1e-4878-89d4-0b65f71587e4.png)

Check:
![Screenshot 2022-11-01 at 19 07 10](https://user-images.githubusercontent.com/65104164/199306251-12b5d817-06e3-483b-b4d9-9791a0f3a1fe.png)

Read w/ status = churned:
![Screenshot 2022-11-01 at 19 16 09](https://user-images.githubusercontent.com/65104164/199308350-385ebe4c-ba31-45be-8fec-8a379ccc9a0b.png)

Read w/ status = active:
![Screenshot 2022-11-01 at 19 18 34](https://user-images.githubusercontent.com/65104164/199308384-f3c4ffff-24e9-47f0-9fc5-3797c105ca79.png)

Read w/ status = activeOrChurned:
![Screenshot 2022-11-01 at 19 26 40](https://user-images.githubusercontent.com/65104164/199309871-75eb0c1d-ff68-49e2-a999-beaac74547c7.png)